productivity-suite/app/fragments/todos/src/components/Header.tsx

```diff
@@ -26,6 +26,8 @@ export function Header({
         }`}
         placeholder="What needs to be done?"
         autoFocus
+        autoComplete="off"
+        spellCheck={false}
```

Comment:
Can you please add a comment with a reference to the chrome big we filed? Thanks